Neos.Fusion/Classes/Core/ObjectTreeParser/Ast/AssignedObjectPath.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Fusion\Core\ObjectTreeParser\Ast;
+
+/*
+ * This file is part of the Neos.Fusion package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Fusion\Core\ObjectTreeParser\AstNodeVisitorInterface;
+
+#[Flow\Proxy(false)]
+class AssignedObjectPath extends AbstractNode
```

Comment:
the `objectPath` naming etc is still present in the ast nodes which i think is okay - since we talk about the fusion object paths (the ones of a file) and not about the merged array for the runtime?
duh this is con**fusion** and makes totally no sense what im doing ^^